Integrations/McAfee_ESM-v10/McAfee_ESM-v10.py

```diff
@@ -288,9 +289,21 @@ def fetch_alarms(self, time_range, custom_start, custom_end, assigned_user):
         if time_range == 'CUSTOM' and (not custom_start or not custom_end):
             return_error('you must specify customStart and customEnd when timeRange is CUSTOM')
```

Comment:
raise ValueError instead return_error - return_error should be done on Exception catch